### ISSUE TITLE
Custom message for Github checks

### ISSUE THREAD

**Issue by antoinegag** [danger/danger-js#897]
I think it would be nice to be able to set the message that is displayed on the GH checks when something goes wrong. 

Would that be possible or that's a Github limitation?

<img width="704" alt="Image 2019-07-22 at 9 39 43 AM" src="https://user-images.githubusercontent.com/25696312/61637004-c0a11500-ac64-11e9-96da-a8dd3c6ae148.png">

**Comment by Soyn** [comment]
You means we can add custom message in config when check is failed or successful?

**Comment by antoinegag** [comment]
Yes, I would like to be able to set it somewhere, possibly in my dangerfile.

**Comment by Soyn** [comment]
In current project code, current hint info is hard code. Maybe we need to figure out a way to do this. But i'm not sure. This feature is necessary for user. @orta

**Comment by robcresswell** [comment]
Ha, I came looking for the same thing. The PR check for us usually just says `Danger: Everything is fine`, which is a pretty useless status compared to something like `CircleCI: E2E Tests - Your tests passed on CircleCI!` as an example; it gives no explanation as to what Danger is, or what it is testing, and from a language PoV having a big green tick next to "Danger" is not very encouraging when I'm merging code!

**Comment by orta** [comment]
It's not really something I'm particularly pumped to support, but if a well tested PR comes in with a way to change it inside the Dangerfile DSL I'm cool with that 👍

**Comment by robcresswell** [comment]
@orta Understood! I appreciate the tooling, but don't have the time to add such a feature. Thanks for the swift response
